pytest_fixtures/component/taxonomy.py

```diff
@@ -76,6 +76,22 @@ def module_org_with_manifest(module_org):
     return module_org
 
 
+@pytest.fixture(scope='module')
+def module_entitlement_manifest_org(module_target_sat, module_entitlement_manifest):
+    """Creates an organization and uploads an entitlement mode manifest generated with manifester"""
+    org = module_target_sat.api.Organization().create()
```

Comment:
I could be wrong on this, but would it make more sense to inherit the `module_org`, since we have other fixtures that will be also inheriting it? Or do we want this to be completely separate? My initial thoughts are for UI testing, we set the default org to `module_org` so in this case it might be more beneficial for that to be the same org.

Reply:
Initially, I was thinking that I should match the `module_manifest_org` fixture, so that's why I used `module_target_sat`. But I agree that it might make more sense to inherit `module_org` instead.

Reply:
I think, it should match the `module_org_with_manifest fixture`, I'm not sure why we have two fixtures for the same thing: `module_manifest_org` and `module_org_with_manifest`

Reply:
@Gauravtalreja1 I have wondered that myself! But ack, just pushed the change. Will trigger PRT again to be sure it's still passing.

Reply:
Either way `module_manifest_org` and `module_org` should be the same organization I believe.

Reply:
@synkd Thanks, you can open a separate PR to consolidate those fixtures later, non-blocking for this PR, will resolve this thread.